src/lava/magma/core/model/py/type.py

```diff
@@ -4,6 +4,12 @@
 import typing as ty
 from dataclasses import dataclass
 from lava.magma.core.model.py.ports import PyInPort, PyOutPort, PyRefPort
+import numpy as np
+import warnings
+
+# Members of LavaPyType needed for float- to fixed-point conversion.
+CONST_CONV_VARS = ('num_bits_exp', 'domain', 'constant', 'scale_domain',
```

Comment:
Conv might be mistaken with convolution. Also, does this constant need to be globally defined? Seems to be used only in conversion_data(..).

Reply:
Addressed in 0117a669a3f2c37e943eefb164a56100d4267f21.